zipkin-junit5/README.md

```diff
@@ -0,0 +1,58 @@
+# zipkin-junit5
+
+This contains `ZipkinRule`, a JUnit rule to spin-up a Zipkin server during tests.
```

Comment:
This doesn't look right. It contains `ZipkinExtension` which is not a JUnit rule.

Reply:
@shakuzen Thanks, fixed